### PR TITLE
fix: special script to build jsdocs

### PR DESCRIPTION
**Pull request by petermuessig**

### CODE REVIEW

**Comment by petermuessig** [comment]
Cc: @matz3 @RandomByte @akudev

**Review by akudev — COMMENTED** [review]
Looks good, but let's wait until it works, I guess.

File: ui5-jsdoc.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+specVersion: "3.0"
+metadata:
+  name: "com.myorg.myui5lib"
+type: library
+builder:
+  customTasks:
+    - name: ui5-tooling-transpile-task
+      afterTask: replaceVersion
+server:
```
Comment (akudev) [inline ui5-jsdoc.yaml]: I only wonder why we need the server part when this is only for the jsdoc build

Comment (petermuessig) [reply ui5-jsdoc.yaml]: The transpiling still needs to happen before the JSDoc generation starts 😉

Comment (akudev) [reply ui5-jsdoc.yaml]: In the `server:` section?

Comment (petermuessig) [reply ui5-jsdoc.yaml]: Ah, I just saw the customTask above. You highlighted the `server:` line - yes, this is useless

Comment (akudev) [reply ui5-jsdoc.yaml]: Yeah, for some reason it also displays the three preceding lines, which are of course perfectly fine.
image

**Pull request merged**